### ISSUE TITLE
[REQ] Option to replace Montserrat and Arimo with Gotham and Arial

### ISSUE THREAD

**Issue by Pagong27Gaming** [pizzaboxer/bloxstrap#2313]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

I think we should have an option to make it so any piece of text in games that is written in the fonts Arimo or Montserrat will have their font changed to Arial or Gotham, since I find these fonts to be utter garbage.

**Comment by pizzaboxer** [comment]
Just those specific fonts only and not a universal font change?

**Comment by Pagong27Gaming** [comment]
> Just those specific fonts only and not a universal font change?

yes

**Comment by pizzaboxer** [comment]
Yeah, I've been thinking about making the ability to replace fonts a lot more flexible like this. I'll definitely do it at some point.
